src/AppBundle/Entity/Classes.php

```diff
@@ -12,6 +12,11 @@
  */
 class Classes
```

Comment:
Singular Entity class name.

Reply:
https://github.com/saadmehmood/school-management-system/blob/470964757bdcb2c757c71c15ee80f74c78fcdd3f/src/AppBundle/Entity/Classes.php#L8

What is 'Class', please add proper doc block